03_letters/CameronSchafer/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+// string to rune with counts
+func letters(s string) map[rune]int {
+	var m = make(map[rune]int)
+
+	for _, r := range s {
+		m[r]++
+	}
+
+	return m
+}
+
+func sortLetters(m map[rune]int) []string {
+	var keys = []int{}
+	for val := range m {
+		keys = append(keys, int(val))
+	}
+
+	// sort the ints
+	sort.Ints(keys)
```

Comment:
A couple of things here. I don't think this comment adds any value, it is self explanatory in the method name what the call is doing.

And secondly, you can eliminate the first iteration over the map, if you directly create a `[]string` as you have below. Then you should be able to directly sort it using `sort.Strings` to achieve the same result with a single iteration.

Can you please update this?

Reply:
Done.
Stumped me for a bit on this one.